e2e/Makefile

```diff
@@ -170,7 +170,7 @@ create-cluster: topolvm.img
 	$(HELM) repo add jetstack https://charts.jetstack.io
 	$(HELM) repo update
 	$(HELM) dependency build ../charts/topolvm/
-	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f $(HELM_VALUES_FILE) $(HELM_SKIP_NODE_FINALIZE_FLAG)
+	$(HELM) install --namespace=topolvm-system topolvm ../charts/topolvm/ -f $(HELM_VALUES_FILE) --set "controller.nodeSelector.kubernetes\.io/hostname"=$(KIND_CLUSTER_NAME)-worker $(HELM_SKIP_NODE_FINALIZE_FLAG)
```

Comment:
Why is only the dot in ".io" escaped with a backslash? Doesn't the dot in "controller.nodeSelector.kubernetes" need to be escaped with a backslash?

Reply:
@cupnes 
The definition of [yaml file](https://github.com/topolvm/topolvm/blob/main/e2e/manifests/values/deployment-scheduler.yaml) is as shown below.
```
controller:
  replicaCount: 1
  securityContext:
    enabled: false
  nodeSelector:
    kubernetes.io/hostname: topolvm-e2e-worker
```
First two dots are used to traverse the yaml tree, but the last dot is just a part of string "kubernetes.io/hostname".

cf. https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set